services/121-service/src/payments/reconciliation/intersolve-visa-reconciliation/intersolve-visa-reconciliation.controller.ts

```diff
@@ -23,12 +25,8 @@ export class IntersolveVisaController {
   })
   @Patch('programs/:programId/financial-service-providers/intersolve-visa/')
   public async retrieveAndUpdateAllCards(): Promise<void> {
-    console.info(
-      'CronjobService - Started: retrieveAndUpdateAllWalletsAndCards',
-    );
-    await this.intersolveVisaService.retrieveAndUpdateAllWalletsAndCards();
-    console.info(
-      'CronjobService - Complete: retrieveAndUpdateAllWalletsAndCards',
-    );
+    console.info('Job started: retrieveAndUpdateAllWalletsAndCards');
```

Comment:
Nice trick, removing the "Cron" :)
Why not just log this in CronjobService.cronRetrieveAndUpdateVisaData()? Because that is where the actual job is. This function does not know anything about a "job".